website/src/pages/handbook/architecture/continuous-integration-testing.mdx

```diff
@@ -3,7 +3,7 @@
 This example demonstrates setting up test coverage for a stitched schema. Like all application code,
 sitched schemas are subject to development errors and therefore should also be tested rigorously
 through a continuous integration pipeline while
-[versioning schema releases](../versioning-schema-releases).
+[versioning schema releases](/handbook/architecture/versioning-schema-releases).
```

Comment:
Do we really need to make these absolute?

Reply:
hmm yeah, we use absolute paths for all our websites